vendor/assets/stylesheets/dvl/components/sidebar_sub_actions.scss

```diff
@@ -5,5 +5,8 @@
   font-size: $fontSmall;
   .sidebar_box + & {
     text-align: right;
+    .dropdown_body {
+      text-align: left;
```

Comment:
Let's add a dropdown to the docs so we can see this without running a separate app?